packages/nextjs/components/Footer.tsx

```diff
@@ -58,17 +58,17 @@ export const Footer = () => {
               </p>
               <a
                 className="flex justify-center items-center gap-1"
-                href="https://buidlguidl.com/"
+                href={projectConfig.creatorLink}
```

Comment:
see some lines above, we have `Built with <HeartIcon className="inline-block h-4 w-4" /> at` and it will work fine with creatorLink only if it a company or smth like that. For me, "https://buidlguidl.com/" is good and shouldn't change